libethereum/ExtVM.cpp

```diff
@@ -33,7 +33,7 @@ namespace // anonymous
 static unsigned const c_depthLimit = 1024;
 
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
-static size_t const c_singleExecutionStackSize = 6 * 1024;
+static size_t const c_singleExecutionStackSize = 13 * 1024;
```

Comment:
This has to be fixed. 12 is way too large. Does it crash in evmjit or in the interpreter?